tests/e2e/hellodapr_test.go

```diff
@@ -93,7 +93,8 @@ func TestHelloDapr(t *testing.T) {
 			}
 
 			// trigger test
-			resp, err = httpPost(fmt.Sprintf("%s/tests/%s", externalURL, tt.testCommand), json.Marshal(cmd))
+			body, _ := json.Marshal(cmd)
```

Comment:
What if there is an error here?

Reply:
Changed the code. I thought that we did not need to check error because cmd is defined in the same code.